### PR TITLE
Add colour and font options documentation

### LINKED ISSUE #418: Documentation on setting colours
It would be nice to include some documentation on how to set terminal colours. In my case I'd like to recreate my urxvt colour theme. I have the following Xresources set:

```
*background:              S_base03
*foreground:              S_base0
*fading:                  40
*fadeColor:               S_base03
*cursorColor:             S_base1
*pointerColorBackground:  S_base01
*pointerColorForeground:  S_base1

*color0:                  S_base02
*color1:                  S_red
*color2:                  S_green
*color3:                  S_yellow
*color4:                  S_blue
*color5:                  S_magenta
*color6:                  S_cyan
*color7:                  S_base2
*color9:                  S_orange
*color8:                  S_base03
*color10:                 S_base01
*color11:                 S_base00
*color12:                 S_base0
*color13:                 S_violet
*color14:                 S_base1
*color15:                 S_base3
```

I've managed to replicate foreground and background colours by setting in `~/.config/termite/config`:

```
[colors]
foreground = #839496
background = #002b36
```

Are there any other colors I can set.

I'd also like to know if it's possible to set my colours in `~/.config/gtk-3.0/gtk.css`, that way I can centralise my colour preferences by defining css variables.

Thanks for the terminal!

### PR DESCRIPTION
**Pull request by silasdavis**
Fixes #418

### CODE REVIEW

**Comment by polyzen** [comment]
Sorry, I don't see the use of including all this in the readme, especially considering Solarized is no longer maintained. (PS: https://github.com/alpha-omega/termite-colors-solarized)

I was unaware of the font fallback bit.. a note for that would be great (maybe even just in the config file).

**Comment by silasdavis** [comment]
Um, it's really nothing to do with Solarized, that was just incidental. The point was to include some documentation somewhere about how to set the colours.

There is no information in the man page or the readme on how to set colours. I just used an exemplar as I thought it would be the quickest way for people to see how to set termite up to work like they probably have it in done via .Xresources.

**Comment by polyzen** [comment]
> I just used an exemplar as I thought it would be the quickest way for people to see how to set up termite to work like they probably have done for other terminals using .Xresources.

I don't know how the maintainers feel about linking to any particular colorscheme. Seems it would be best to include this documentation in the termite.config man page.

> termite doesn't generate a config file

I meant https://github.com/thestinger/termite/blob/master/config

File: README.rst [hunk 1]
```diff
@@ -158,6 +158,45 @@ completions, escape closes the widget and enter accepts the input.
...
+    color10 = #586e75
+    color11 = #657b83
+    color12 = #839496
+    color13 = #6c71c4
+    color14 = #93a1a1
+    color15 = #fdf6e3
+    # ... and up to color<255> = #<rgb hex value>
+
+Colours in exemplar are taken from Solarized dark (http://ethanschoonover.com/solarized)
+
```
Comment (jelly) [inline README.rst]: I don't have the feeling that defining a colour scheme here adds much. The colors also differ from the default config.

The other options are already in the config with added comments. For the color values see:
https://github.com/thestinger/termite/blob/master/config#L50

**Comment by silasdavis** [comment]
Let's not obsess over the fact I've used a colour scheme! There are some random colours in the config file, so we can just as well make no reference to where the colours come from, I just wanted to see them documented. As you point out they are in the config file. But that is not linked from the readme and I didn't find that (I guess I dismissed it as being part of the build or something, call me dense, but I'm probably not the only one). 

So perhaps It's worth putting a link to the commented config file at the very top of the README where it talks about where the file is, but doesn't describe its format or properties. Personally I find information about setting the colours sufficiently interesting to go in the README given the other stuff that is there. But a link to the documented config file would work too.

**Comment by jelly** [comment]
the README describes the config in the first section, by default the config file is installed in /etc/xdg/termite/config.

The current config describes more about the colors (for example highlight) then this PR. Also it's duplicating content which I rather not have since it's more work.

Tools like https://terminal.sexy/ are more suited for generating a color scheme.

**Comment by jelly** [comment]
Sorry, I don't see what this PR adds what's not in the man page / readme.

**Pull request closed without merging**